src/components/vm/overview/vmOverviewCard.jsx

```diff
@@ -244,7 +244,7 @@ class VmOverviewCard extends React.Component {
                                             </Flex>
                                         }
                                         hasAutoWidth>
-                                        <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                        <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230704-095439-c26f64f8-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)